module/module.go

```diff
@@ -559,3 +739,18 @@ func writeFile(filePath string, bytes []byte) error {
 func closeFile(file *os.File) {
 	_ = file.Close()
 }
+
+var camelingRegex = regexp.MustCompile("[0-9A-Za-z]+")
+
+func camelCase(src string) string {
```

Comment:
It causes a circular reference and there's no clear place to share the functionality. That's why I copied it. I'm assuming we can resolve this in future  by moving module code into the codegen package at some point. TBD.